telegram/ext/jobqueue.py

```diff
@@ -56,6 +72,12 @@ def __init__(self, bot, prevent_autostart=None):
         """:type: float"""
         self._running = False
 
+    def timedelta_to_seconds(self, tdelta):
+        try:
+            return tdelta.total_seconds()
+        except AttributeError:
+            raise AttributeError("timedelta_to_seconds only takes a datetime.timedelta object")
+
     def put(self, job, next_t=None):
         """Queue a new job.
 
```

Comment:
Please update the docstring so it mentions that `next_t` can also be of type `timedelta`